robot/rover/Arm/Vsense.h

```diff
@@ -14,13 +14,6 @@ void vbatt_read() {
 
   UART_PORT.print("ARM battery voltage: ");
   UART_PORT.println(vbatt);
-
-  if (vbatt < 12) {
-    UART_PORT.println("ARM WARNING! BATTERY VOLTAGE IS LOW! DISCONNECT IMMEDIATELY!");
-  }
-  else if (vbatt > 16.8) {
-    UART_PORT.println("ARM WARNING! BATTERY VOLTAGE IS HIGH! DISCONNECT IMMEDIATELY!");
-  }
```

Comment:
Why is this being removed?

Reply:
Do you want me to comment it out instead? You told me we should remove this
spam since it was super cluttering.

Reply:
Also with the GUI, I can't see a situation where these logs would be useful.

If you're connected to a Teensy on your laptop, then you don't have the battery --> This is useless

If your teensy is connected to the rover, and the battery is low --> You won't see these messages but the GUI will tell you --> This is useless

Reply:
so it seems to me there's no point to even comment it out

Reply:
I added them back, in a discussion in other channels. We agreed that this was out of scope